examples/gdn/example_chunk_delta_h.py

```
@@ -94,15 +113,15 @@ def tilelang_chunk_gated_delta_rule_fwd_h(
     state_dtype,
     chunk_size,
-    use_g=True,
-    use_initial_state=True,
-    store_final_state=True,
-    save_new_value=True,
+    use_g,
+    use_initial_state,
+    store_final_state,
+    save_new_value,
     # kernel config
     block_DK=64,
```

Ruff (0.14.4)

Unused function argument: `block_DK`

(ARG001)